File: Codeforces/1. ICPC Assiut University Community/Contest #1/A. Winter Sale/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/A
#include<iostream>
#include<cstdio>

using namespace std;
 
int main(){
    unsigned short x;
    unsigned int p;
    cin >> x >> p;
    x = 100 - x;
    float result = ((float)p/x) * 100;
    printf("%0.2f", result);
}

File: Codeforces/1. ICPC Assiut University Community/Contest #1/A. Winter Sale/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/A

using System;

namespace Codeforces.Assuit.Contest1
{
    class ProblemA
    {
        static void Main(string[] args)
        {
            string[] input = Console.ReadLine().Split(' '); // take one line input
            ushort x = ushort.Parse(input[0]);
            uint p = uint.Parse(input[1]); 

            x = (ushort)(100 - x);
            float result = ((float)p/x) * 100;
            Console.WriteLine(result.ToString("F2"));
        }
    }
}
